react-user-table/src/components/features/users/AddUsers.tsx

```diff
@@ -0,0 +1,128 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addUser } from "../redux/UserReducer";
+import { useNavigate } from "react-router-dom";
+import { RootState } from "../../../main";
+
+export type UserProps = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  phone: string;
+};
+
+export function AddUsers() {
+  const [name, setName] = useState("");
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
```

Comment:
Mega szkoda że nie wdrożyłaś react-hook-form, aktualne rozwiązanie nie dodaje zbytnio punktów w rekrutacji

Reply:
Swoją drogą, jeżeli nie ma dodawania userów to nie potrzebujemy tego kodu